### PR TITLE
K-ary Heap

### PR DESCRIPTION
**Pull request by kanisshka**
Closes : #1852

### CODE REVIEW

**Comment by kanisshka** [comment]
@Rohan27s 
please check this...

File: Code/C++/k_ary_heap.cpp [hunk 1]
```diff
@@ -0,0 +1,214 @@
+// C++ program to demonstrate all operations of
+// k-ary Heap
```
Comment (Rohan27s) [inline Code/C++/k_ary_heap.cpp:2]: Add description here

File: Code/C++/k_ary_heap.cpp [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+	printf("\n\nHeap after extract max: \n");
+	for (int i=0; i<n; i++)
+		printf("%d ", arr[i]);
+
+	return 0;
+}
+
+
+
+/*
```
Comment (Rohan27s) [inline Code/C++/k_ary_heap.cpp:164]: Mention the Space and time complexity

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Review by Rohan27s — APPROVED** [review]
Nice Work! Thanks for your Contribution

**Pull request merged**